packages/components/upload/__tests__/upload-dragger.test.tsx

```diff
@@ -1,45 +1,43 @@
-import { computed, defineComponent, h, provide } from 'vue'
+import { computed, defineComponent, provide } from 'vue'
+import { mount } from '@vue/test-utils'
 import { describe, expect, test, vi } from 'vitest'
-import makeMount from '@element-plus/test-utils/make-mount'
 import { uploadContextKey } from '@element-plus/tokens'
 import UploadDragger from '../src/upload-dragger.vue'
 
 const AXIOM = 'Rem is the best girl'
 
-const Wrapper = defineComponent({
-  props: {
-    onDrop: Function,
-  },
-  setup(props, { slots }) {
-    provide(uploadContextKey, { accept: computed(() => 'video/*') })
-    return () => h(UploadDragger, props, slots)
-  },
-})
-const mount = makeMount(Wrapper, {
-  slots: {
-    default: () => AXIOM,
-  },
-})
+const _mount = (options = {}) =>
+  mount(
+    defineComponent({
+      setup() {
+        provide(uploadContextKey, { accept: computed(() => 'video/*') })
+      },
+      render() {
+        return <UploadDragger {...this.$attrs}>{AXIOM}</UploadDragger>
+      },
+    }),
+    options
+  )
```

Comment:
Maybe we can simplify the mount function like this:

```suggestion
const _mount = (props = {}) =>
  mount({
    setup() {
      provide(uploadContextKey, { accept: computed(() => 'video/*') })
      return () => <UploadDragger {...props}>{AXIOM}</UploadDragger>
    },
  })
```

Reply:
@holazz Thanks you for the review!